src/java.base/share/classes/sun/security/validator/SymantecTLSPolicy.java

```diff
@@ -154,18 +157,22 @@
     static void checkDistrust(X509Certificate[] chain)
                               throws ValidatorException {
         X509Certificate anchor = chain[chain.length-1];
-        if (FINGERPRINTS.contains(fingerprint(anchor))) {
+        String fp = fingerprint(anchor);
+        if (fp != null && FINGERPRINTS.contains(fp)) {
```

Comment:
I understand the original behavior is also bypassing the check if fingerprint cannot be calculated, but this sounds a little irresponsible. Same as in `UntrustedCertificates`.

Reply:
Right, I see your point, and I agree it makes more sense in this case to fail, so I will change it. However, if this situation were to occur because of a rogue X509Certificate impl, it is also possible for it to return an encoding that generates a different fingerprint.